### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by Khaleelur-Rahman**
Added screenshots to all relavant features.

Added Table of Contents.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -147,85 +176,85 @@ Format: `find_contact KEYWORD [MORE_KEYWORDS]`
...
-* An event must have mandatory fields: name, date and address
 * The event name **must not already exist** in JobFestGo.
-* Date must be valid and should be in the appropriate (YYYY-MM-DD) format
-* Date should **not** be before current date
+* Date must be valid and should be in the appropriate (YYYY-MM-DD) format.
+* Date should **not** be before the current date.
 
 Examples:
-* `add_event n/NUS Career Fest 2023 d/2023-12-23 a/NUS` adds an event named `NUS Career Fest 2023` to JobFestGo.
+* `add_event n/CS2103T Presentataion d/2023-11-10 a/311, Clementi Ave 2, #02-25` adds an event named `CS2103T Presentation` to JobFestGo. <br><br>
```
Comment (rionshocker) [inline docs/UserGuide.md:245]:
```suggestion
* `add_event n/CS2103T Presentation d/2023-11-10 a/311, Clementi Ave 2, #02-25` adds an event named `CS2103T Presentation` to JobFestGo. <br><br>
```

Comment (rionshocker) [reply docs/UserGuide.md:245]: Just a typo

File: docs/UserGuide.md [hunk 2]
```diff
@@ -259,9 +287,9 @@ Format: `link ev/EVENT_NAME c/CONTACT_NAME [c/MORE_CONTACT_NAMES]`
  c/John Doe c/Bob Dylan` will link `Alice Black` to the event `NUS Career Fest` while
   `John Doe` and `Bob Dylan` will not be linked.
 
-### Selecting an event: `select_event`
+### Selecting an event : `select_event`
 
-Selects a specified event from JobFestGo. The relevant contacts and tasks will be displayed.
+Selects a event at the specified index from JobFestGo. The relevant contacts and tasks will be displayed.
```
Comment (rionshocker) [inline docs/UserGuide.md:292]:
```suggestion
Selects an event at the specified index from JobFestGo. The relevant contacts and tasks will be displayed.
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -282,12 +312,17 @@ Format: `add_task td/TASK_DESCRIPTION d/DEADLINE ev/EVENT_NAME`
...
-* `add_task td/Book Venue d/2023-12-23 ev/NUS Career Fair 2023` adds a `Book Venue` by `2023-12-23` task to the event `NUS Career Fair 2023`
+* `add_task td/Book Venue d/2023-12-23 ev/NUS Career Fair 2023` adds a `Book Venue` by `2023-12-23` task to the event `NUS Career Fair 2023`.
+
+### Viewing help : `help`
+
+Shows a message explaining how to access the help page.
 
+![help message](images/helpMessage.png)
+
+Format: `help`
```
Comment (rionshocker) [inline docs/UserGuide.md:325]: Can include a spacing between this line and the next line

**Review by rionshocker — CHANGES_REQUESTED** [review]
Apart from some minor fixes needed, LGTM!

File: docs/UserGuide.md (hunk 1 above)
Comment (Khaleelur-Rahman) [reply docs/UserGuide.md]: Good catch. Thanks!

File: docs/UserGuide.md (hunk 3 above)
Comment (Khaleelur-Rahman) [reply docs/UserGuide.md:325]: Done!

File: docs/UserGuide.md (hunk 2 above)
Comment (Khaleelur-Rahman) [reply docs/UserGuide.md]: Good catch too!

**Pull request merged**